src/main/java/se/sundsvall/supportmanagement/integration/messaging/configuration/MessagingConfiguration.java

```diff
@@ -0,0 +1,25 @@
+package se.sundsvall.supportmanagement.integration.messaging.configuration;
+
+import org.springframework.cloud.openfeign.FeignBuilderCustomizer;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Import;
+import org.springframework.security.oauth2.client.registration.ClientRegistrationRepository;
+
+import se.sundsvall.dept44.configuration.feign.FeignConfiguration;
+import se.sundsvall.dept44.configuration.feign.FeignMultiCustomizer;
+import se.sundsvall.dept44.configuration.feign.decoder.ProblemErrorDecoder;
+
+@Import(FeignConfiguration.class)
+public class MessagingConfiguration {
+
+    public static final String CLIENT_REGISTRATION_ID = "messaging";
```

Comment:
Very minor:

Maybe just CLIENT_ID? It's used in clientRegistrationRepository (where the name makes sence), but it's also used in circuitbreaker an ErrorDecoder (and maybe some other place?).

CLIENT_ID is a more generic name i.m.h.o.